README.md

```
@@ -1,96 +1,150 @@
+git clone https://github.com/your-username/too-many-buttons.git
+cd too-many-buttons
+
 
 # Run
-[commands]
+# Option 1: Open directly
+open index.html         # macOS / Linux
+start index.html        # Windows
+
 
 ### Project Documentation
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)